### PR TITLE
Notes

### PR DESCRIPTION
**Pull request by alxmrtnz**
Initial comments added to SessionModel and client.js

### CODE REVIEW

File: client/src/js/models/SessionModel.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    this.updateSessionUser(this.user.defaults);
+
+    // Reset the SessionModel's values to defaults and show 
+    // that a user is now logged out.
+    this.set({ userId: this.user.get('id'), logged: false });
+  },
+
+  updateSessionUser: function(userData) {
+
+    // A user model is passed into this function.
```
Comment (akrawchyk) [inline client/src/js/models/SessionModel.js]: Not quite, the user model itself is not passed, just the data that we received from the authentication to be saved in the user model.

File: client/src/js/models/SessionModel.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    console.log('auth2 is signed in: ', auth2.isSignedIn.get());
+
+    // If a User is signed in and their gmail matches a @isl.co gmail,
+    // log in this user.
+    if (userSignedIn && profile.getEmail().match(/^.*@isl.co$/g)) {
+      this.loginSessionUser(googleUser);
+      if ('success' in callback) {
+
+        // If user is successfully signed in, call back the 
+        // auth2SignInChanged()'s success function (in client.js)
```
Comment (akrawchyk) [inline client/src/js/models/SessionModel.js]: One place this is called is in client.js. We might need to use this callback in other places too in the future :+1:

**Comment by alxmrtnz** [comment]
@akrawchyk modified those two notes that you called out. Check them over and I'll merge it all into client? or master?

**Comment by akrawchyk** [comment]
:+1: merge it into master

**Pull request merged**